recipe/migrations/libopenvino_dev2024.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for libopenvino_dev 2024
+  kind: version
+  migration_number: 1
+libopenvino_dev:
+- '2024'
```

```suggestion
- '2024.0.0'
```

why did the pin from the feedstock get ignored, they all seem to pin at `x.x.x`
https://github.com/conda-forge/openvino-feedstock/blob/main/recipe/meta.yaml